pkg/scripts/os_amzn.go

```diff
@@ -165,9 +163,10 @@ sudo yum install -y --disableplugin=priorities \
 {{- end }}
 	kubernetes-cni \
 	cri-tools
-sudo yum versionlock add kubelet kubeadm kubectl kubernetes-cni cri-tools
 {{- end }}
 
+sudo yum versionlock add kubelet kubeadm kubectl kubernetes-cni cri-tools
```

Comment:
Why is this line moved? If we remove the lock only if the Kubernetes repo are used, why do we add the lock even if the Kubernetes repo are not used?

_note: this variable is always `true` at the moment, but this might change very soon._

Reply:
So that it always adds version lock back, regardless of conditions. it's like a defer.